### ISSUE TITLE
Android ARM 64

### ISSUE THREAD

**Issue by jsaade** [rizasif/sqlite-unity-plugin#3]
Hello,

The package is missing sqlite compatible with android arm-64 as Google set it as a requirement now to build 64bits native apps.

I tried adding a libesqlite3.so 64 bits that I found online but it did not work.

Please advise.

**Comment by rizasif** [comment]
Hey,

I haven't looked into that issue. What kind of problems do you get while trying the 64bit version?

**Comment by jsaade** [comment]
Here is the error:
` E/Unity: DllNotFoundException: Unable to load DLL 'sqlite3': The specified module could not be found.
      at Mono.Data.Sqlite.UnsafeNativeMethods.sqlite3_open_v2 (System.Byte[] utf8Filename, System.IntPtr& db, System.Int32 flags, System.IntPtr vfs) [0x00000] in <00000000000000000000000000000000>:0 
      at Mono.Data.Sqlite.SQLite3.Open (System.String strFilename, Mono.Data.Sqlite.SQLiteOpenFlagsEnum flags, System.Int32 maxPoolSize, System.Boolean usePool) [0x00000] in <00000000000000000000000000000000>:0 
      at Mono.Data.Sqlite.SqliteConnection.Open () [0x00000] in <00000000000000000000000000000000>:0 
      at SQLiteManager.openConnection () [0x00000] in <00000000000000000000000000000000>:0 
      at SQLiteManager.Awake () [0x00000] in <00000000000000000000000000000000>:0 `

**Comment by rizasif** [comment]
I am not sure about this issue.

Please feel free to submit a pull request if you find anything :)

**Comment by jsaade** [comment]
I am trying, the sqlite lib included:
Plugins/Android/libs/armeabi-v7a/libsqlite3.so

did you compile it or get it from somewhere? The reason is that we need another .so for 64 bits

**Comment by rizasif** [comment]
Hi,

I believe since Unity builds in the windows space it won't be possible to include an ARM version. However, Since a 64 bit version is already included in the repository so you won't have to worry about the google requirement.

Hope this helps.

**Comment by ByMedion** [comment]
@rizasif Hello! I think this issue needs to be reopened. Repository has 64-bit version, but its not linked to ARM64 apk because of ARM64 library required :(
ARM64 library from sqlite.org does not works, also official ARMv7 (762kb) version has difference with yours (327kb). Can you share source of your version of libs?

**Comment by ByMedion** [comment]
Solution found! I just picked the ARM64 library from here: https://github.com/mapbox/mapbox-unity-sdk/blob/develop/sdkproject/Assets/Mapbox/Core/Plugins/sqlite/Android/libs/arm64-v8a/libsqlite3. So
I also updated Mono.Data.SQLite.dll from 2.0.0.0 to 4.0.0.0 (but I'm not sure if this is necessary).
Sadly this doesnt work on 2019.4, only 2019.3 (not checked at other virsions). Maybe problem in some different code stripping algorithm, need to check.
